node-scripts/utils.js

```diff
@@ -3,6 +3,35 @@ const slugify = require('slugify');
 const toSlug = (path) =>
   slugify(path, { lower: true, trim: true }).replace('.', '-');
 
+const specialCasing = {
+  p5js: 'p5.js',
+  p5: 'p5.js'
+}
+
+const getFormattedWord = (word) => {
+  const getLookupKey = (word) => {
+    return word.replace(/[!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~]/g, '').toLowerCase();
+  }
+
+  if (specialCasing[getLookupKey(word)]) {
+    return specialCasing[getLookupKey(word)]
+  }
```

Comment:
OK. This was really random, but I think I found the issue.

So the error you were getting was when trying to create all pages for the site. When trying to get all loaded language and topic tags, the query crashed because there was some tag somewhere loaded as a `null`. I had a hard time founding it, but I realized that the `"constructor"` tag got transformed by `cleanUp` to a function instead of another string. Which seem VERY weird.  

But the funny thing is that because of how you were checking if the tag had a special casing, that specific word is a reserved property for all objects (a method actually), so it actually returned said method. So, to fix it, it should be as simple as changing how you check if a tag need special casing. I added a suggestion that worked locally for me.

```suggestion
  if (specialCasing.hasOwnProperty(getLookupKey(word))) {
    return specialCasing[getLookupKey(word)]
  }
```